cmd/vela-slack/main.go

```diff
@@ -66,6 +72,11 @@ func main() {
 			Usage:    "set log level - options: (trace|debug|info|warn|error|fatal|panic)",
 			Value:    "info",
 		},
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_SSL_CERT_FILE", "SSL_CERT_FILE"},
```

Comment:
i see there is no `FilePath` defined for this flag. is that on purpose?

Reply:
The certification path get's defined in the dockerfile, but i'm not opposed to including a filepath if people want to override it?